### ISSUE TITLE
Make driver license information more visible (and more flexible?)

### ISSUE THREAD

**Issue by MichiK** [engelsystem/engelsystem#567]
Angels can enter their driver license information in their profile and angeltypes can be configured to require a driver license. However, the system already has different typs of licenses from car to truck to forklift but there is no way to restrict angel types or shifts to holders of a specific type of license. In addition, there is no way that allows shift coordinators to view and filter angels by driver license.

During 35C3 buildup, we had several requests for truck drivers but there was no easy way to get to this information.

In addition, I think it would be good to transform this into a more abstract concept of special skills or licenses people might have, like "Steigerschein" or "Frikadellendiplom" or "Unterweisung für die Bedienung des Lichtschalters in Halle 3".

**Comment by cedi** [comment]
Agree with the overall idea. Would be very helpful for @c3loc.
But i think we need some extra offline validation in place, so people from the heaven or @c3loc can check the License in question and then approve it.

I think this could be already archived by adding an restricted angel type "Forklift Driver". So if you want to play with the Forklift you have to sign-up for that angel type, go to the @c3loc with your License to be  approved for this restricted angel type.
By doing it this way we also collect less personal information in the engelsystem

**Comment by der-michik** [comment]
Of course, simple requirements for a single license can be handled via angel types.

However, how would you handle e.g. a shift where huge amounts of food need to be transported that requires a truck driver with a Frikadellendiplom?
